### PR TITLE
Update image and copy for the "Contribute" page

### PR DESCRIPTION
**Pull request by GBKS**
This takes the content from the README in the Meta repo and puts it in the website as a regular page. I made some copy tweaks and added a header image, see issue #53 for more details.

The page so far has just had some placeholder lorem ipsum. My main goal with this PR is to get a decent V1 ready that we can iterate on.

Contribute page 201209

### CODE REVIEW

**Comment by pavlenex** [comment]
@GBKS .@ConorOkus mentioned that the How to contribute to Bitcoin Design Guide video series I've made may be suitable to be inserted into this page. I'm not sure if you plan a sub-page or perhaps we can add a [link to the playlist](https://www.youtube.com/playlist?list=PLPZzDjXV0FTZwILz4uyqiJNjUX-kHWP0k)?

Can do it in a separate PR if you're waiting for #53 first.

**Comment by GBKS** [comment]
@pavlenex great idea. Feel free to add it to this PR. I need to review and clean up some links in this page anyways (for example, the "Project intro" link still points to a Google Doc).

I could see this become a sub-page at some point. How about we start by linking to the playlist?

File: contribute.md [hunk 1]
```diff
@@ -1,53 +1,81 @@
...
+1. See [projects](Projects.md) for a few project ideas
+1. Contribute to the [Bitcoin Design Guide](https://github.com/BitcoinDesign/Guide)
+
+## Bitcoin Design Guide
+
+- Project Intro — [Doc v2.0.0](https://docs.google.com/document/d/1YiYeRIybGmxmErCOI4Jc8Qajz3JGM1JYVfUtpzyCzSk/edit?usp=sharing)
+- Slack Channel — [#bitcoin-design-guide](https://bitcoindesign.slack.com/archives/C015856BDME)
+- Github Repository — [Guide](https://github.com/BitcoinDesign/Guide)
+- Github Project Board — [Guide/Content](https://github.com/BitcoinDesign/Guide/projects/1)
+
```
Comment (pavlenex) [inline contribute.md:54]:
```suggestion
```suggestion
[How To Contribute to Bitcoin Design Guide video series](https://www.youtube.com/playlist?list=PLPZzDjXV0FTZwILz4uyqiJNjUX-kHWP0k)

```

```

File: contribute.md [hunk 2]
```diff
@@ -1,53 +1,81 @@
...
+## Learn how we work
+
+If you are not familiar with the concepts of open design, we have a [great intro]({{ '/guide/getting-started/open-design/' | relative_url }}) for you. Open, public collaboration gives a lot of freedom to each participant and relies on us organically coming up with the organizational structures we want. The result is that things can sometimes look unorganized or unintuitive from the outside. The big benefit is that you can choose how you want to participate. Let's jump in.
+
+## How to get involved in the community
+
+1. [Join the community on Slack](https://join.slack.com/t/bitcoindesign/shared_invite/zt-gytq2snl-4TEWJOTKrXRCB4YLBoDunA)
+2. [Subscribe to the newsletter](https://bitcoindesign.substack.com) to stay up-to-date
+2. Read up on our [project life cycle](Projects.md)
```
Comment (pavlenex) [inline contribute.md:36]: Broken link, needs lowercase `p` `projects.md`
```suggestion
2. Read up on our [project life cycle](projects.md)
```

File: contribute.md [hunk 3]
```diff
@@ -1,53 +1,81 @@
...
+3. Browse [issues](https://github.com/BitcoinDesign/Meta/issues) for upcoming calls and discussions around processes and coordination
+4. Subscribe to our [calendar](https://raw.githubusercontent.com/BitcoinDesign/Meta/calendar/events.ical) for calls and events we organize
+
+## How to get involved in projects
+
+1. See the projects we are either involved in or recommend on the [projects]({{ 'projects' | relative_url }}) page
+1. Browse [issues labelled as projects](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aopen+is%3Aissue+label%3Aproject)
+2. Browse and post ideas in the [#project-ideas](https://bitcoindesign.slack.com/archives/C0174N5KUF9) Slack channel
+1. See [projects](Projects.md) for a few project ideas
```
Comment (pavlenex) [inline contribute.md:45]:
```suggestion
1. See [projects](projects.md) for a few project ideas
```

File: contribute.md [hunk 4]
```diff
@@ -1,53 +1,81 @@
...
+1. [Join the community on Slack](https://join.slack.com/t/bitcoindesign/shared_invite/zt-gytq2snl-4TEWJOTKrXRCB4YLBoDunA)
+2. [Subscribe to the newsletter](https://bitcoindesign.substack.com) to stay up-to-date
+2. Read up on our [project life cycle](Projects.md)
+3. Browse [issues](https://github.com/BitcoinDesign/Meta/issues) for upcoming calls and discussions around processes and coordination
+4. Subscribe to our [calendar](https://raw.githubusercontent.com/BitcoinDesign/Meta/calendar/events.ical) for calls and events we organize
+
+## How to get involved in projects
+
+1. See the projects we are either involved in or recommend on the [projects]({{ 'projects' | relative_url }}) page
+1. Browse [issues labelled as projects](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aopen+is%3Aissue+label%3Aproject)
```
Comment (pavlenex) [inline contribute.md:43]: Typo
```suggestion
1. Browse [issues labeled as projects](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aopen+is%3Aissue+label%3Aproject)
```

**Comment by pavlenex** [comment]
Ping @GBKS can you try to resolve the conflicts in contribute.md ,seems that's the only blocker now.

**Comment by GBKS** [comment]
@pavlenex just did.

**Comment by danielnordh** [comment]
ACK. Looks good to me and much better than nothing!
Merge ahead.

**Pull request merged**